README.md

```diff
@@ -139,4 +139,4 @@ Feel free to check the [issues page](https://github.com/mahdinoori2000/blog-app/
 
 ## 📝 License <a name="license"></a>
 
```

Comment:
- [ ] It would be better if we add the command `rspec spec` under the `Run Tests` section header since this is a testing project, it would be better to add it to make the README file more professional